notebooks/samples/explanations/ai-explanations-image-tf2.ipynb

```diff
@@ -0,0 +1,1758 @@
+{
```

Comment:
<p>(Remove heading, replace with sentence)</p><p>Send the explanation request for the baseline image:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='89'/>